src/agent.cpp

```diff
@@ -162,6 +165,14 @@ int Agent::getUnum() {
   return world().self().unum();
 }
 
+void Agent::setLastActionStatusCollision(bool may_fix, bool likely_success) {
```

Comment:
I'd like to get rid of this method and instead just have each of the actions return an action status:

switch(requested_action) {
      case DASH:
          last_action_status = this->doDash(...);
          break;
      case MOVE:
}

Hopefully it should be possible to put the correct logic about what status to return in each of the behaviors.